### PR TITLE
Add CEReactions tests for some HTML elements

### PR DESCRIPTION
**Pull request by xiuqijix**
- Covers elements of HTMLSourceElement/HTMLStyleElement/HTMLTableCellElement/HTMLTableColElement

part of issue #5727

### CODE REVIEW

**Comment by kaixinjxq** [comment]
@domenic, PTAL.

**Comment by kaixinjxq** [comment]
@domenic, friendly ping :-)

File: custom-elements/reactions/HTMLSourceElement.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```
Comment (domenic) [inline custom-elements/reactions/HTMLSourceElement.html]: Since this file introduces new helpers, can you separate it out into a separate PR, so that someone can review the helpers in more detail?

**Review by domenic — COMMENTED** [review]
LGTM for the ones besides HTMLSourceElement. HTMLSourceElement is more complicated and I'm unsure whether I'll be able to make time to review it in detail, so let's do that in a separate PR.

**Review by domenic — APPROVED** [review]
